dot/sync/chain_sync.go

```diff
@@ -336,26 +335,17 @@ func (cs *chainSync) logSyncSpeed() {
 			cs.benchmarker.end(after.Number.Uint64())
 			target := cs.getTarget()
 
-			logger.Info("🔗 imported blocks", "from", before.Number, "to", after.Number,
-				"hashes", fmt.Sprintf("[%s ... %s]", before.Hash(), after.Hash()),
-			)
-
-			logger.Info("🚣 currently syncing",
-				"peer count", len(cs.network.Peers()),
-				"target", target,
-				"average blocks/second", cs.benchmarker.mostRecentAverage(),
-				"overall average", cs.benchmarker.average(),
-				"finalised", finalised.Number,
-				"hash", finalised.Hash(),
-			)
+			logger.Infof(
+				"🔗 imported blocks from %d to %d (hashes [%s ... %s ])",
```

Comment:
```suggestion
				"🔗 imported blocks from %d to %d (hashes [%s ... %s])",
```

Reply:
How did my own eyes miss this NOOoooo 😄 👀